tests/console/clamav.pm

```diff
@@ -40,6 +40,12 @@ sub run {
     $self->select_serial_terminal;
 
     zypper_call('in clamav vim');
+
+    # Check Clamav version
+    # Jira ticket : SLE-16780 : upgrade Clamav SLE
+    zypper_call('info clamav');
+    record_info 'Check Clamav Version';
+
```

Comment:
I think the perfect usage for record_info is like, e.g.,
./hpc/hpc_migration.pm:    record_info('INFO: current version', "$current_version");

Reply:
do you mean like this I write originally ?
```
    my $current_ver = script_output("zypper se -s clamav");
    record_info("Clamav_ver", "Current Clamav package version : $current_ver");
```

Reply:
yes

Reply:
I think both way are fine here, let me think about it. List the version and record_info the version will be the better one. Thanks.